src/main/kotlin/org/rust/lang/core/psi/impl/RustRawStringLiteralImpl.kt

```diff
@@ -4,6 +4,15 @@ import com.intellij.psi.tree.IElementType
 import org.rust.lang.core.psi.RustLiteralTokenType
 
 class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustTextLiteralImplBase(type, text) {
+    private var _hashes: Int = 0
+
+    val hashes: Int
+        get() {
+            @Suppress("UNUSED_VARIABLE")
+            val o = offsets // ensure #computeOffsets() has been called
```

Comment:
> One more thing to keep in mind: offsets for "" (an empty string) have null as value, so the value of this string literal is null instead of "". 

Fixed, where was the issue spotted?